test/langtools/jdk/javadoc/doclet/testLinkOption/pkg/B.java

```diff
@@ -53,14 +53,14 @@ public void createTempFile(String s1, String s2, File f){}
    /**
     * A external links, complicated by whitespace
     * @see <a href="http://www.ietf.org/rfc/rfc2279.txt"><i>RFC&nbsp;2279: UTF-8, a
-    * transformation format of ISO 10646</i></a>, <br><a
-    * href="http://www.ietf.org/rfc/rfc2373.txt"><i>RFC&nbsp;2373: IPv6 Addressing
-    * Architecture</i></a>, <br><a
-    * href="http://www.ietf.org/rfc/rfc2396.txt"><i>RFC&nbsp;2396: Uniform
-    * Resource Identifiers (URI): Generic Syntax</i></a>, <br><a
-    * href="http://www.ietf.org/rfc/rfc2732.txt"><i>RFC&nbsp;2732: Format for
-    * Literal IPv6 Addresses in URLs</i></a>, <br><a
-    * href="C.html">A nearby file</a>
+    * transformation format of ISO 10646</i></a>
+    * @see <a href="http://www.ietf.org/rfc/rfc2373.txt"><i>RFC&nbsp;2373: IPv6 Addressing
+    * Architecture</i></a>
+    * @see <a href="http://www.ietf.org/rfc/rfc2396.txt"><i>RFC&nbsp;2396: Uniform
+    * Resource Identifiers (URI): Generic Syntax</i></a>
+    * @see <a href="http://www.ietf.org/rfc/rfc2732.txt"><i>RFC&nbsp;2732: Format for
+    * Literal IPv6 Addresses in URLs</i></a>
+    * @see <a href="C.html">A nearby file</a>
     */
```

Comment:
Wow, I didn't know we had "bad" examples in the test code.   If there isn't one already, can you file an issue for me to updated DocLint with a check for this?